rundeckapp/grails-app/views/common/_navData.gsp

```diff
@@ -119,11 +119,11 @@
 
                     <g:if test="${params.project ?: request.project}">
                         <g:ifMenuItems type="PROJECT" project="${projectName}">
-                            <g:forMenuItems type="PROJECT" var="item" project="${projectName}">
+                            <g:forMenuItems type="PROJECT" var="item" project="${projectName}" groupvar="group">
                 {
                     type: 'link',
                     id: 'nav-${item.title.toLowerCase().replace(' ', '-')}-link',
-                    group: 'main',
+                    group: '${enc(attr:group?.id?:'main')}',
```

Comment:
@ProTip I looked at the NavBar.vue which seems to hardcode only 'main' and 'bottom' groups.  So I wasn't sure whether I should add this change here.  The GroupedMenuItem allows some grouping behavior in the System Config menu (also includes a label for the group), however the dropdown menu has different usage than the project side navbar, not sure if dynamic group name should be enabled here